package.json

```diff
@@ -21,7 +21,7 @@
     "color": "^4.0.1",
     "date-fns": "^2.24.0",
     "debounce-promise": "^3.1.2",
-    "dompurify": "^2.5.4",
+    "dompurify": "^3.2.4",
```

Comment:
Why did you update to 3.2.4 instead of updating to the latest 2.5 version, which is [2.5.8](https://www.npmjs.com/package/dompurify/v/2.5.8)?

Reply:
It doesn't fix the underlying dependabot XSS issue and v3 removes IE support which we do not support.

Reply:
See also: https://github.com/cure53/DOMPurify/releases